webapp/src/components/Modal/AuctionModal/AuctionModal.js

```diff
@@ -3,6 +3,10 @@ import PropTypes from 'prop-types'
 import { Checkbox, Button } from 'semantic-ui-react'
 import { t, T } from '@dapps/modules/translation/utils'
 
+import {
+  hasAgreedToTerms as hasAgreedToOtherTerms,
+  agreeToTerms as agreeToOtherTerms
```

Comment:
_Silly comment_ Maybe instead of `Other` we could use something like `Global` or `Main`? I.e `hasAgreedToMainTerms`